src/Application/Models/Asset.js

```diff
@@ -1,7 +1,7 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
+  constructor(name, price, multiplier = 1, owned = 0, unlocked = false, investment = 0, revenue = 0) {
```

Comment:
I don't feel strongly about it belonging to one or the other, but having the defaults in the constructor makes it a little more understandable imo. Also, I really like how clean it makes stateutils.